utils/scan_utils.py

```diff
@@ -452,7 +452,7 @@ def build_scan_options_parser() -> ArgumentParser:
     parser.add_argument("--sslyze-reneg",
                         help="sslyze: If set, will use the SessionRenegotiationScanner and return session renegotiation info. Defaults to true.")
     # trustymail:
-    parser.add_argument("--starttls", help="".join([
+    parser.add_argument("--starttls", action='store_true', help="".join([
```

Comment:
I don't think we want `store_true` for these four arguments.  `store_true` creates a default value, but we really only care if the argument is present or not.  See [here](https://docs.python.org/2/library/argparse.html#action).

This is what is breaking some of the tests.

Reply:
Without `store_true` there are unnecessary extra arguments required (e.g., `--mx MX` where just `--mx` would be sufficient) since they are really just switches, but if it helps compatibility, I'll put them back.

Reply:
OK, that makes sense.  I will put it back the way you had it and fix the tests instead.